### PR TITLE
Change URLs to not open in a new tab

### LINKED ISSUE #67: Change all links so they open in the same tab
These days I believe it's best practice to open hyperlinks in the same tab.

The user can choose to open them in a new tab if they wish.

Opening in a new tab can also be a security risk.

### PR DESCRIPTION
**Pull request by prestonhunter**
Removing target="_blank" rel="noopener"

fixes #67

### CODE REVIEW

File: _includes/barnabus/link.html [hunk 1]
```diff
@@ -1 +1 @@
- <a href="https://www.upwork.com/fl/johnbampton" target="_blank" rel="noopener">Barnabus</a>
\ No newline at end of file
+ <a href="https://www.upwork.com/fl/johnbampton"  >Barnabus</a>
```
Comment (jbampton) [inline _includes/barnabus/link.html]: There should be no space inside the HTML `a` tag at the end of the tag.  

In some cases you added an extra space.  So you may need to remove 2 spaces from some lines

File: flags.html [hunk 2]
```diff
@@ -15,10 +15,10 @@
     <br>
     <p>
         <a href="{{w}}{{ row.country }}"
-            target="_blank" rel="noopener">{{ row.country  }}</a>
+             >{{ row.country  }}</a>
```
Comment (jbampton) [inline flags.html]: Better to make this `a` tag into one line

File: flags.html [hunk 3]
```diff
@@ -15,10 +15,10 @@
...
     </p>
     <a href="{{w}}{{ row.country }}"
-        target="_blank" rel="noopener">
+         >
```
Comment (jbampton) [inline flags.html]: Better to make this `a` tag into one line

File: index.html [hunk 4]
```diff
@@ -36,7 +36,7 @@ <h2>
         {% endif %}
     </div>
     <br><br>
-    <p class="rss-subscribe">Subscribe via<a href="{{ "/feed.xml" | prepend: site.baseurl }}" target="_blank"> RSS </a> or
-        <a href="{{ "/atom.xml" | prepend: site.baseurl }}" target="_blank">Atom</a>
+    <p class="rss-subscribe">Subscribe via<a href="{{ "/feed.xml" | prepend: site.baseurl }}"  target="_blank"> RSS </a> or
+        <a href="{{ "/atom.xml" | prepend: site.baseurl }}"  target="_blank">Atom</a>
```
Comment (jbampton) [inline index.html]: Need to remove ` target="_blank"` from both lines

**Comment by jbampton** [comment]
Hey @prestonhunter I am going to take over this pull request and finish it.

I am going to squash the commits.

**Comment by prestonhunter** [comment]
Thank you for letting me know.

**Comment by jbampton** [comment]
@all-contributors add @prestonhunter for code

**Pull request merged**